Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM openjdk
+FROM openjdk:8-jdk-alpine
 WORKDIR /
-ADD target/conter-0.0.1-SNAPSHOT.jar conter-0.0.1-SNAPSHOT.jar
+COPY target/conter-0.0.1-SNAPSHOT.jar app.jar
 EXPOSE 8080
-CMD java -jar conter-0.0.1-SNAPSHOT.jar
+ENTRYPOINT [ "sh", "-c", "java -Dutopia.search.API=$1 -Dutopia.booking.API=$2 -Dutopia.cancellation.API=$3 -Djava.security.egd=file:/dev/./urandom -jar /app.jar" ]
```

Comment:
What does the `java.security.egd` property do? (And its value should be just `/dev/urandom`, if that file is correct; the `./` does absolutely nothing.)

Reply:
we set up system property “java.security.egd” which will override the securerandom.source setting. to avoiding JVM delays caused by random number generation in the running phase .

Reply:
(Never mind about the `./` doing nothing; I've now found that it's needed to work around a Java misfeature.) Are we actually seeing such delays? The one place I immediately found in the Spring documentation mentioning this property it was in reference to running on one's local machine (not even running the JAR!) via the `run` Maven (or Gradle) task.

Reply:
actually we are not seeing any delays. it's just a prevention measure